core/base/src/main/java/alluxio/uri/URI.java

```diff
@@ -96,6 +96,9 @@ public static URI create(String uriStr) {
 
       // uri path is the rest of the string -- fragment not supported
       String path = uriStr.substring(start, uriStr.length());
+      if (!uriStr.equals("") && path.equals("")) {
```

Comment:
What is the reason to check `uriString.equals("")` vs only checking `path.equals("")`?

Reply:
if the given path is "", we don't want to add a "/" to path, which cause the uri to be "/" instead of ""